### PR TITLE
Document local RelayHub in README

### PR DESCRIPTION
**Pull request by PaulRBerg**
For #5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,19 @@ const getDeploymentAndFunds = async () => {
...
+When you run `npx oz fund-recipient`, the following [code](https://github.com/OpenZeppelin/openzeppelin-gsn-helpers/blob/master/src/fund.js) gets executed:
+
+```js
+// Ensure relayHub is deployed on the local networkNicolás Venturo, 1 day ago: • Remove fund into own script
+if (options.relayHubAddress.toLowerCase() === data.relayHub.address.toLowerCase()) {
+  await deployRelayHub(web3, options.from);
+}
+```
+
+Note that both on mainnet and in testning environments, the address of the `RelayHub` contract is always `0xD216153c06E857cD7f72665E0aF1d7D82172F494`.
```
Comment (nventuro) [inline README.md]:
```suggestion
Note that both on mainnet, testnets and local blockchain (`ganache`) environments, the address of the `RelayHub` contract is always the same: `0xD216153c06E857cD7f72665E0aF1d7D82172F494`.
```

**Comment by nventuro** [comment]
[`gsn-helpers`](https://github.com/OpenZeppelin/openzeppelin-gsn-helpers) also includes a `deploy-relayhub` command, which will just do the deployment, but we attempted to make the process more seamless by having it be done automatically on the other scripts, since it is an idempotent action. 

Could you share a bit more regarding which parts of the process struck you as confusing? We're still reviewing our guides, helpers, etc., and it'd be great to take all of this feedback to improve them.

**Comment by PaulRBerg** [comment]
Whenever I learn a new thing, I go very deep to understand it.

Since the RelayHub is the core piece of infrastructure (regardless of its idempotent status), I want to be fully aware of the deployment process. I'll be writing tests, and it's likely that these tests will touch the hub somehow. But even if they don't, I still want to ensure I fully understand the contract, because ultimately I will wire money for my users through it.

**Comment by nventuro** [comment]
That makes a lot of sense, and I'm glad you're looking at this in detail :)

The other reason why we made the deployment automatic is to reinforce the fact that you don't need to deploy your own instance of the hub on real networks: having to do so in local blockchains seemed to cause confusion unnecessarily.

**Pull request merged**